pkg/question/question.go

```diff
@@ -121,6 +115,18 @@ func AskQuestion(input *AskQuestionInput) string {
 
 		// No match at all
 		fmt.Println("Input invalid. Please try again.")
+		GetQuestion(input)
+	}
+}
+
+// Displays question with default values
```

Comment:
It's standard go style to start func documentation/comments with func name, ex: 

`// GetQuestion displays question with default values`

Reply:
I didn't knew it earlier as I had seen the comments for other functions and thought explanation would work, thanks for the information, will modify this as well :)